### PR TITLE
Add MySQL, Gradle, SDKMAN

### PR DESCRIPTION
**Pull request by Elyssen-1**
Added help about the Database and SDKMAN! for easier Gradle 4.8 install.

### CODE REVIEW

**Comment by Elyssen-1** [comment]
Well Adyne went dumb. Changed it a bit more in 13509e7

File: linux-guide.md [hunk 1]
```diff
@@ -49,26 +51,77 @@ Start by logging into your server via your SSH client.
...
+    git --version
 
 If the command is not found you can install git by running
 
     sudo apt install git
 
+<a name="install-mysql"></a>
+## Installing MySQL
+
+We'll need to setup a database that Ava can use. Ava requires a database to store things like custom playlists, aliases, channel and server data, and a lot of other things, it currently only has support for MySQL databases, with SQLite and other types comming in the future.
```
Comment (Senither) [inline linux-guide.md:90]: SQLite is supported, I might've just not updated the documentation to reflect that yet.

File: linux-guide.md [hunk 2]
```diff
@@ -82,19 +135,17 @@ You have now successfully cloned the AvaIre Git repository, next we'll need to b
 
     gradle build
 
-If the build finishes with no errors you should be good to go! You can find the generated binary file at `build/libs/AvaIre.jar`. 
+If the build finishes with no errors you should be good to go! You can find the generated binary file at `build/libs/AvaIre.jar`.
```
Comment (Senither) [inline linux-guide.md:138]: Ava now uses a shadow build process so the jar file is actually placed in the same folder as the `gradle.build` file is located in. (My mistake, I guess I don't update the documentation enough)

**Review by Senither — COMMENTED** [review]
There are some minor things that isn't correct, it's completely my fault for being lazy and not updating the documentation enough, if you wanna write documentation for it that would be amazing, otherwise I'll do it after your PR is merged, everything else looks good^^ 👍

**Pull request merged**